### ISSUE TITLE
recognize album dates

### ISSUE THREAD

**Issue by legacychimera247** [nate-xyz/resonance#74]
i have an artist called american football, they made 3 records and all of them are called "american football", reasonance as of now shows me only one of the records but when i click on the play buttons all get added to the queue, anyway to solve this?

![Schermata del 2023-04-13 13-33-09](https://user-images.githubusercontent.com/22254556/231746338-5837967b-8910-4435-8774-9b7b779eb115.png)
![Schermata del 2023-04-13 13-33-26](https://user-images.githubusercontent.com/22254556/231746358-4e5fa1c1-c36f-4dff-8538-7c4ecca67300.png)

**Comment by DWW256** [comment]
It would seem that solving this issue is mutually exclusive to solving issue #52, unless someone knows of a special `albumdate` tag or something that is commonplace.  If not all tracks in an album always have the same year, how do we determine which album is which?

**Comment by legacychimera247** [comment]
Don't really know, but tauon music box had a similar problem and solved it, maybe it would be nice to have a look at it :)

also lollypop manages those 3 album just fine, so maybe a look at?

**Comment by DWW256** [comment]
Mmm, here's a thought—do you have MusicBrainz AcoustIDs in the tags for these?

**Comment by legacychimera247** [comment]
Yep, althought if i recall, they solved the problem looking at the exact date, may be wrong anyway eh
